keen/explorer
# Fixes #116: Double timeframes in API query URL

## What does this PR do? How does it affect users?

It ensures the api query URL is constructed properly. There was a bug where absolute timeframe queries built an API query URL with multiple timeframe properties. This fixes that issue.

## How should this be tested?

1) Try creating a query with an absolute timeframe.
2) Run it.
3) Check the API query URL and ensure it's correct.
4) Change the absolute timeframe and repeat a few times.

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:


## Related tickets?
#116
